packages/pass-style/test/prepare-test-env-ava.js

```diff
@@ -1,5 +1,7 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I was curious why I seemed to need these in marshal but not in pass-style. So I looked at it in the IDE and see that I do seem to need it here too. `yarn lint` isn't the problem. Rather, it is vscode:

<img width="696" alt="image" src="https://user-images.githubusercontent.com/273868/212236200-d31dcd9e-2e1e-471b-b392-0eec1b77798e.png">

Reply:
If we truly do need these lines into into prepare-test-env-ava.js, they should come from the scaffold, not this PR.

Reply:
VSCode is showing an eslint error. `yarn lint` should show the same. It isn't?

Reply:
> VSCode is showing an eslint error. yarn lint should show the same. It isn't?

It was not.

Reply:
No longer a problem. Resolving.